.circleci/config.yml

```diff
@@ -19,16 +19,16 @@ jobs:
           paths:
             - .bundle
       - restore_cache:
-          key: v1-pods-{{ checksum "Podfile.lock" }}
+          key: v2-pods-{{ checksum "Podfile" }}
       - run:
           name: Install CocoaPods
           command: |
-            curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
-            pod install
+            [ -d "Pods" ] || (curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf; pod install)
       - save_cache:
-          key: v1-pods-{{ checksum "Podfile.lock" }}
+          key: v2-pods-{{ checksum "Podfile" }}
           paths:
             - Pods
+            - Podfile.lock
```

Comment:
do you happen to have a reference for this change by any chance ? I worked on that with the lead engineer from circleci and I wonder if there was a reason dantoml used the .lock file over the Podfile.

Reply:
I don't have any references but I noticed that after running `pod install`, the `PODFILE CHECKSUM` variable within `Podfile.lock` changes each time. Because of this, the cache-key used to save the cache is different than that used when loading the cache.

Oh but I just noticed this 😅
> If you are using CocoaPods, then we recommend that you check your Pods directory into source control. This will ensure that you have a deterministic, reproducible build.
https://circleci.com/docs/2.0/testing-ios/#using-cocoapods